submissions/antonkottans/memory-pair-game/style.css

```diff
@@ -0,0 +1,93 @@
+* {
+    margin: 0px;
+    box-sizing: border-box;
+}
+.game-heading {
+    padding: 20px;
+}
+
+.header {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+}
+.result {
```

Comment:
Be consistant with empty lines, place one between each block for better readability.